packages/bumbag-addon-highlighted-code/jest.config.js

```diff
@@ -1,6 +1,6 @@
-const { ...baseConfig } = require('../../jest.config');
+const { ...baseConfig } = require("../../jest.config");
 
 module.exports = {
   ...baseConfig,
-  snapshotSerializers: ['jest-emotion']
+  snapshotSerializers: ["@emotion/jest"],
```

Comment:
Looks like all the changed files are being replaced with double quotes. Can we run `yarn lint` to fix this?

Reply:
Will do, seems like my global prettier config changing that, thoughts on adding a .prettierrc/prettier to the codebase?

Reply:
We already have our prettier config in our ESLint config (https://github.com/bumbag/bumbag-ui/blob/master/.eslintrc#L9). So if you're using the ESLint extension for VSCode, you should be able to do `cmd-shift-p` -> `ESLint: Fix all auto-fixable problems`